gtests/net/mptcp/dss/mpc_with_data_server.pkt

```diff
@@ -24,4 +24,6 @@
 // read 100 bytes from the MPC+data
 +0.01 read(4, ..., 100) = 100
 +0    close(4) = 0
++0      > . 1001:1001(0) ack 101 <dss dack8=101 dsn8=1001 ssn=0 dll=1 nocs fin, nop, nop>
++0      < F. 101:101(0) ack 1001 win 225 <dss dack8=1001 nocs>
```

Comment:
could we here receive a DATA+FIN in this FIN+ACK? (just to make sure it is still accepted or maybe we should add a dedicated test for that?)

Reply:
I'd be surprised to see a TCP FIN here without dack8=1002 and a DATA_FIN. As discussed in the top-level pull request conversation, it's fine if the full handshake is tested in another folder